content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -422,6 +439,15 @@ However, configuring the proxy this way does require cluster provider specific k
 In addition to that, you loose monitoring of access to external services and cannot apply Istio features on traffic to
 external services.
 
+The third approach directs the traffic through the Istio sidecar proxy, however it allows access to any service on any
+ports that have no HTTP service inside the mesh and no service entry defined. In this approach, similarly to the
+previous one, you cannot monitor access to external services. The advantage of this approach is that you don't need to
```

Comment:
With this approach, like the previous one, you ...

Reply:
Addressed in https://github.com/istio/istio.io/pull/3222/commits/be148adab523d7a6b74db00580c28d71b255ec46: With this approach, like **with** the previous one

Reply:
There are two previous approaches, I left a suggestion to be more specific and avoid confusion.